README.md

```diff
@@ -0,0 +1,99 @@
+# Portfolio Project
+
+> This is a project on building my personal portfolio
```

Comment:
Can we be a little more descriptive about the project? What does it do and how does it do it? 🤔